crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -919,15 +955,15 @@ impl GetBatchData for MeshPipeline {
 impl GetFullBatchData for MeshPipeline {
     type BufferInputData = MeshInputUniform;
 
-    fn get_batch_input_index(
+    fn get_index_of_batch_input(
```

Comment:
Index of doesn’t change the meaning. If it only got the index then the old name is better. But it gets the batch data and the index. That’s why I made the suggestion. The binned one below only gets the index so its name was fine. I think this one should be index_and_…

Reply:
I renamed `get_index_of_batch_input` to `get_index_and_compare_data` and also renamed `get_index_of_binned_batch_input` to `get_binned_index`.